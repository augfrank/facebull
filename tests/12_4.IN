M0   C0   C1  2556
M1   C0   C3  2294
M2   C0   C5  2281
M3   C0   C7  2311
M4   C0   C8  2316
M5   C0   C10  2604
M6   C0   C11  2611
M7   C1   C0  2495
M8   C1   C6  2852
M9   C1   C7  2056
M10   C1   C8  2997
M11   C1   C9  2338
M12   C2   C0  2990
M13   C2   C1  2475
M14   C2   C4  2939
M15   C2   C8  2506
M16   C2   C9  2670
M17   C2   C11  2621
M18   C3   C1  2954
M19   C3   C4  2078
M20   C3   C5  2090
M21   C3   C6  2700
M22   C4   C1  2400
M23   C4   C2  2974
M24   C4   C3  2098
M25   C4   C6  2185
M26   C4   C9  2216
M27   C4   C10  2851
M28   C4   C11  2300
M29   C5   C0  2951
M30   C5   C6  2199
M31   C5   C7  2487
M32   C5   C8  2585
M33   C5   C10  2374
M34   C5   C11  2982
M35   C6   C1  2379
M36   C6   C2  2831
M37   C6   C3  2865
M38   C6   C4  2770
M39   C6   C5  2397
M40   C6   C7  2989
M41   C6   C10  2277
M42   C7   C0  2494
M43   C7   C1  2351
M44   C7   C2  2734
M45   C7   C3  2247
M46   C7   C6  2912
M47   C8   C0  2713
M48   C8   C2  2644
M49   C8   C5  2462
M50   C8   C6  2760
M51   C8   C9  2007
M52   C8   C10  2085
M53   C9   C0  2169
M54   C9   C4  2715
M55   C9   C8  2190
M56   C9   C11  2601
M57   C10   C1  2035
M58   C10   C7  2249
M59   C10   C11  2476
M60   C11   C2  2480
M61   C11   C3  2296
M62   C11   C4  2835
M63   C11   C8  2478
M64   C11   C10  2344
