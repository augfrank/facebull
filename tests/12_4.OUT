26635
2 9 15 19 28 30 41 45 51 53 57 60
